include/uprotocol-cpp/transport/datamodel/UListener.h

```diff
@@ -1,10 +1,9 @@
 #ifndef _ULISTENER_H_
 #define _ULISTENER_H_
 
-#include <include/uri/up_uri.h>
-#include <include/uri/up_uri_factory.h>
-#include <include/model/attributes.h>
-#include "UStatus.h"
+#include <uprotocol-cpp/uri/datamodel/UUri.h>
+#include <uprotocol-cpp/transport/datamodel/UAttributes.h>
```

Comment:
UUri, UAttributes, UMessage, UPayload are all defined in proto now, are you pulling them in?

Reply:
@stevenhartley - Yes , on a later stage though 
Adi is working now on the UURi, 
Can we merge without them and do it in stages?

Reply:
yes